core/trino-main/src/main/java/io/trino/operator/scalar/IpAddressFunctions.java

```diff
@@ -85,7 +85,8 @@ public static boolean contains(@SqlType(StandardTypes.VARCHAR) Slice network, @S
         }
 
         if (base.length != ipAddress.length) {
-            throw new TrinoException(INVALID_FUNCTION_ARGUMENT, "IP address version should be the same");
+            return false;
```

Comment:
Please leave a code comment why returning false is fine from CIDR RFC perspective.

Reply:
I have remove this.   I really appreciate the review.  I did not expect it to be looked over so soon or while in draft form.